torchani/models.py

```diff
@@ -282,3 +282,19 @@ def ANI1ccx(periodic_table_index=False, model_index=None):
     if model_index is None:
         return BuiltinEnsemble._from_neurochem_resources(info_file, periodic_table_index)
     return BuiltinModel._from_neurochem_resources(info_file, periodic_table_index, model_index)
+
```

Comment:
@IgnacioJPickering You recently changed this file, so please review

Reply:
The function LGTM, *but* ANI 1x and ANI 1ccx have 339 files each, why does this PR modify 557 files? @cdever01

Reply:
There are a few problems with doc strings in these new changes

Reply:
Yes you are right. @cdever01 told me that the new files come from the new atom types so that should be fine